torchci/components/JobLinks.tsx

```diff
@@ -88,9 +88,9 @@ function getTestName(failureCapture: string) {
   return null;
 }
 
-function formatDisableTestBody(failureCaptures: string) {
+function formatDisableTestBody(failureCaptures: string[]) {
   const examplesURL = `http://torch-ci.com/failure/${encodeURIComponent(
-    failureCaptures
+    failureCaptures[0]
```

Comment:
is there a reason why this and the one above it are [0] instead of the entire array?

Reply:
I correct the type declaration of `failureCaptures` from `JobData` from a string to `string[]`.  Although we use `failureCaptures` here for the more like this feature on HUD, it's basically just an array with one item, thus it's easier to just use the first failure.

Reply:
I read the SQL https://console.rockset.com/lambdas/details/commons.failure_samples_query.  The values here are used in this statement `AND job.torchci_classification.line LIKE FORMAT('%{}%', REGEXP_REPLACE(:captures, ',', '%'))`, so a comma-separated string would work.